plugins/versionlock.py

```diff
@@ -60,26 +60,28 @@ def sack(self):
         if not locklist_fn:
             raise dnf.exceptions.Error(NO_LOCKLIST)
 
-        locked = set()
+        excludes_query = self.base.sack.query().filter(empty=True)
+        include_query = self.base.sack.query().filter(empty=True)
+
         for pat in _read_locklist():
             excl = False
             if pat and pat[0] == '!':
                 pat = pat[1:]
                 excl = True
 
             subj = dnf.subject.Subject(pat)
-            pkgs = subj.get_best_query(self.base.sack)
+            pkgs = subj.get_best_query(self.base.sack, with_nevra=True, with_provides=False,
```

Comment:
According to documentation, versionlock handles only packages by nevra. It excludes packages and not packages based by provides. I disable it because I think it can provide inconsistency in result. What do you think?